test/e2e/ps_test.go

```diff
@@ -896,4 +930,169 @@ var _ = Describe("Podman ps", func() {
 		Expect(session.OutputToString()).To(Or(Equal(net1+","+net2), Equal(net2+","+net1)))
 	})
 
+	// This test checks ps filtering of external container by container command/entrypoint
+	It("podman ps filter external by container command", func() {
+		opts := storage.StoreOptions{
+			ImageStore: podmanTest.ImageCacheDir,
+			RunRoot:    podmanTest.RunRoot,
+			GraphRoot:  podmanTest.Root,
+		}
+
+		storage, err := storage.GetStore(opts)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		builder, err := buildah.NewBuilder(context.TODO(), storage, buildah.BuilderOptions{FromImage: ALPINE, Container: "top"})
+		Expect(err).ShouldNot(HaveOccurred())
```

Comment:
Do not use buildah directly as library in the test code, the test suite is not setup up for this the potential reexec logic can cause quite big problem here. Please stick to podman build and then just kill the process to leak a build container, there should be already examples it the existing --external tests

Reply:
Good to know!
Let me rewrite this particular test.

Reply:
Hello @Luap99 
I couldn't find examples you mentioned in you previous message.

Reply:
I added test which improve overall filter tests. At least it shows no side effect was brought by my test